File: mbw/src/main/java/com/mycelium/wallet/modularisation/GooglePlayModuleCollection.java
package com.mycelium.wallet.modularisation;


import android.content.Context;

import com.mycelium.modularizationtools.model.Module;
import com.mycelium.wallet.BuildConfig;
import com.mycelium.wallet.R;

import java.util.HashMap;
import java.util.Map;

public class GooglePlayModuleCollection {
    public static Map<String, Module> getModules(Context context) {
        Map<String, Module> result = new HashMap<>();
        result.put("bch", new Module("com.mycelium.module.spvbch"
                + (BuildConfig.FLAVOR.equals("btctestnet") ? ".testnet" : "")
                + (BuildConfig.DEBUG ? ".debug" : "")
                , context.getString(R.string.bitcoin_cash_module)
                , context.getString(R.string.bch_module_description)));

        return result;
    }
}

File: mbw/src/main/java/com/mycelium/wallet/modularisation/GooglePlayModuleCollection.kt
package com.mycelium.wallet.modularisation

import android.content.Context
import android.graphics.drawable.Drawable
import com.mycelium.modularizationtools.model.Module
import com.mycelium.wallet.BuildConfig
import com.mycelium.wallet.R
import com.mycelium.wallet.WalletApplication
import com.mycelium.wapi.wallet.bch.bip44.Bip44BCHAccount


object GooglePlayModuleCollection {
    @JvmStatic
    fun getModules(context: Context): Map<String, Module> =
            hashMapOf()

    @JvmStatic
    fun getModuleByPackage(context: Context, packageName: String) =
            getModules(context).values.first { it.modulePackage == packageName }

    @JvmStatic
    fun getBigLogos(context: Context): Map<String, Drawable> =
            hashMapOf()

    @JvmStatic
    fun getBigLogo(context: Context, packageName: String) =
            getBigLogos(context).get(packageName)
}